packages/ts-api-react/src/storage.ts

```
@@ -62,7 +62,16 @@ export class StorageManager {
         return null;
       }
     } catch (e) {
-      console.warn(`JSON parse from ${this.namespace} storage failed`, e);
+      console.warn(`Failed to read or parse value from ${this.namespace} storage`, e);
```

ESLint

[error] Replace ``Failed·to·read·or·parse·value·from·${this.namespace}·storage`,·e` with `⏎········`Failed·to·read·or·parse·value·from·${this.namespace}·storage`,⏎········e⏎······`

(prettier/prettier)